Filter/ModelFilter.php

```diff
@@ -134,4 +134,30 @@ protected function association(ProxyQueryInterface $queryBuilder, $data)
 
         return array($alias, false);
     }
+
+    /**
+     * Retrieve the parent alias for given alias.
+     * Root alias for direct association or entity joined alias for association depth >= 2.
```

Comment:
So... if I'm filtering on `entity.parentEntity`, I'll get the alias for entity, and if I'm filtering on `entity.parentEntity.field` or `entity.parentEntity.field.someOtherField`, I will get the alias for `parentEntity`, is that right?